docs/source/tutorials/transfer_learning/transfer_learning.rst

```diff
@@ -0,0 +1,62 @@
+Using Syne Tune for Transfer Learning
+======================================
+
+Transfer learning allows us to speed up our current optimisation by learning from related optimisation runs. For instance, imagine we want to change from a smaller to a larger model. We already have a collection of hyperparameter evaluations for the smaller model. Then we can use these to guide our hyperparameter optimisation of the larger model, for instance by starting with the configuration that performed best. 
+
+Syne Tune includes implementations of several transfer learning schedulers; a list of available schedulers is given  `here <../../getting_started.html#supported-hpo-methods>`_. In this tutorial we look at three of them: 
+
+* :class:`~syne_tune.optimizer.baselines.ZeroShotTransfer`: First we calculate the rank of each hyperparameter configuration on each previous task. Then we choose configurations in order to minimise the sum of the ranks across the previous tasks. The idea is to speed up optimisation by picking configurations with high ranks on previous tasks.
+* :class:`~syne_tune.optimizer.schedulers.transfer_learning.BoundingBox`: We construct a smaller hyperparameter search space by taking the minimum box which contains the optimal configurations for the previous tasks. The idea is to speed up optimisation by not searching areas which have been suboptimal for all previous tasks.
+* Quantiles (:class:`~syne_tune.optimizer.schedulers.transfer_learning.quantile_based.quantile_based_searcher`): We map the hyperparameter evaluations to quantiles for each task. Then we learn a distribution of quantiles given hyperparameters. Finally we sample from the distribution and evaluate the best sample. The idea is to speed up optimisation by searching areas with high-ranking configurations but without enforcing hard limits on the search space.
+
+We compare them to standard :class:`~syne_tune.optimizer.baselines.BayesianOptimization` (BO). We construct a set of tasks based on the height example. We first collect evaluations of five tasks, and then compare results on the sixth. We consider the single-fidelity case.
```

Comment:
I'd explain here how these different tasks are constructed, and also other things, such as how many evals are done for each, and whether they are done at random (i.e., random search), or with (say) BO

Reply:
I've added a bit here, and it gets explained in more detail later.